packages/webapp/components/Proposal/BaseProposals.tsx

```diff
@@ -0,0 +1,123 @@
+import {
+  Heading,
+  VStack, Divider, Text, HStack, Stack, Button
+} from "@chakra-ui/react"
+import { FC } from "react"
+import Link from "next/link"
+import { getQuorumVotes, parseState } from "@utils/governanceUtils.ts"
+import { ProposalCard } from "@components/Governance/ProposalCard.tsx"
+import { UserVotes } from "@components/Governance/Delegation/UserVotes.tsx"
+import { DelegateButton } from "@components/Governance/Delegation/DelegateButton.tsx"
+import { isArray } from "lodash"
+import {
+  ProposalState
+} from "@data/contract/requests/getProposalState.ts"
+import { Proposal } from "@queries/base/requests/proposalQuery.ts"
+
+interface BaseProposalsProps {
+  proposals: Proposal[] | undefined
+}
+
+const BaseProposals: FC<BaseProposalsProps> = ({ proposals }) => {
+  if (!isArray(proposals)) return null
+
+  const isFinalized = (state: ProposalState) => state !== ProposalState.Active && state !== ProposalState.Queued
+  const activeProposals = proposals?.filter(({ state }) => !isFinalized(state)) || []
+  const inactiveProposals = proposals?.filter(({ state }) => isFinalized(state)) || []
+  return (
+    <VStack w={"full"} spacing={4} alignItems={"center"} py={{ base: 4, lg: 10 }} px={{ base: 4, lg: 20 }}>
+      <Stack
+        direction={{ base: "column", sm: "row" }}
+        w="full"
+        justifyContent={{ base: "center", sm: "space-between" }}
+        alignContent={{ base: "center", sm: "end" }}
+      >
+        <Heading as={"h2"} fontSize="5xl">
+          Base Proposals
+        </Heading>
+        <Stack direction={{ base: "column", sm: "row" }} alignItems={{ sm: "center" }}>
+          {/*<UserVotes mr={{ sm: 4 }} fontSize={"xl"} fontWeight={"bold"} /> TODO: Delegation Ticket needed */}
+          <Link href={"https://nouns.build/dao/base/0x880fb3cf5c6cc2d7dfc13a993e839a9411200c17?tab=activity"}>
+            <Button w="full">New Proposal</Button>
+          </Link>
+          {/*<DelegateButton /> TODO: Delegation Ticket needed */}
+        </Stack>
+      </Stack>
+      <VStack w={"full"} spacing={4} alignItems={"center"} py={{ base: 4, lg: 20 }} px={{ base: 4, lg: 20 }}>
+        <HStack w="full" pb={10}>
+          <Divider />
+          <Heading as={"h3"} fontSize="4xl">
+            ACTIVE
+          </Heading>
+          <Divider />
+        </HStack>
+        {activeProposals.map(prop => (
+          <Link
+            key={"active-base-prop-" + prop.proposalNumber}
+            href={`https://nouns.build/dao/base/0x880fb3cf5c6cc2d7dfc13a993e839a9411200c17/vote/${prop.proposalNumber}`}
+            style={{ width: "100%" }}
+          >
+            <ProposalCard
+              id={String(prop.proposalNumber)}
+              title={prop.title as string}
+              titleProps={{ noOfLines: 2 }}
+              status={parseState(prop.state).toUpperCase()}
```

Comment:
I was wanted to do that too but at the same time I was thinking "do our end users really care" if we don't have perfect `.ts` everywhere. It leads you down a TypeScript rabbit hole b/c nouns.build has a LOT of types that are similar yet different than ours and imo it's a lot of wasted time rn as long as the fnctionality is there

Reply:
At the end of the day it doesn't matter. I'll defer to you on the effort:value ratio on this, but I am personally of the opinion that once you **start** cutting corners, they'll continue getting cut.

Reply:
I'm of the same opinion on this. There's a ratio for sure and this is just displaying data, but they very much will care when someone's vote gets dropped, they can't vote in the proposal time frame because of a type or error check wasn't considered or the entire proposal gets disrupted or errors out when the outcome is important. 

Approving for this one, but for the proposal data and functionality that needs to be working at 100% we can't be cutting corners or it would make more sense to just redirect them to nouns.